ATOMS.md

```diff
@@ -0,0 +1,63 @@
+## Mobiledoc Atoms
+
+Atoms are effectively read-only inline cards.
+
+An atom is an object with a name and at least one of several hooks defined. For example:
+
+```js
+var demoAtom = {
+  name: 'mention',
+  display: {
+    setup(element, options, env, text, payload) {
```

Comment:
I know this is annoying, but let's change this to follow the new card schema as suggested in https://github.com/bustlelabs/mobiledoc-kit/issues/228#issuecomment-156578655

```js
// package: mda-profile-dom
export default {
  name: 'profile',
  type: 'dom',
  render({element, options, env, text, payload}) {
  }
};
```